Python/bltinmodule.c

```diff
@@ -2019,7 +2019,7 @@ builtin_min(PyObject *self, PyObject *const *args, Py_ssize_t nargs, PyObject *k
 }
 
 PyDoc_STRVAR(min_doc,
-"min(iterable, *[, default=obj, key=func]) -> value\n\
+"min(iterable, /, *[, default=obj, key=func]) -> value\n\
```

Comment:
This is not a valid Python syntax.  Same for max().

Reply:
I understand the issue being that the square brackets make it invalid syntax, and that without this change, the documentation would still be wrong.

The reason for change here is to ensure that the web docs align with the code docs, similar to sorted:

https://github.com/python/cpython/blob/5a3f13b4f4d43cb67232cd8994b49df65b296ffd/Python/bltinmodule.c#L2575

Would it be more correct to instead write:

```suggestion
"min(iterable, /, *, key=None) -> value\n\
min(iterable, /, *, default, key=None) -> value\n\
min(arg1, arg2, *args, /, *, key=None) -> value\n\
```

Reply:
@skirpichev Could you please help me understand why `-> value` should be removed?

Reply:
Rationale is just same as for sphinx docs: it's not a valid syntax (you will get a NameError trying to parse such signature with `inspect.signature()`).

Reply:
> I understand the issue being that the square brackets make it invalid syntax, and that without this change, the documentation would still be wrong.

Actually, it's a valid Sphinx syntax. It's the "legacy" way of writing signatures. Many built-ins are written that way. As for `->`, I think it was meant for "readability" even if it's unparsable. I would personally leave this untouched in this PR (namely, only add positional-only marker but leave out the `->`)